libs/ui-watt/src/lib/components/checkbox/watt-checkbox.component.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+watt-checkbox {
+  --watt-checkbox-disabled-color: var(--watt-color-neutral-grey-400);
+  --watt-checkbox-color: var(--watt-color-primary);
+  --watt-checkbox-checkmark-color: var(--watt-on-dark-high-emphasis);
+
+  display: block;
+
+  input {
+    appearance: none;
+    margin: 0;
+    padding: 0;
+    width: 1.15em;
+    height: 1.15em;
+    border: 0.15em solid var(--watt-checkbox-color);
+    display: grid;
+    place-content: center;
+  }
+
+  input::before {
+    content: "";
+    width: 0.65em;
+    height: 0.65em;
+    transform: scale(0);
+    transition: 120ms transform ease-in-out;
+    box-shadow: inset 1em 1em var(--watt-checkbox-checkmark-color);
+    transform-origin: bottom left;
+    clip-path: polygon(14% 44%, 0 65%, 50% 100%, 100% 16%, 80% 0%, 43% 62%);
+  }
+
+  input:checked::before {
+    transform: scale(1);
+  }
+
+  input:checked {
+    background-color: var(--watt-checkbox-color);
+  }
+
+  input:disabled {
+    border-color: var(--watt-checkbox-disabled-color);
+    background-color: var(--watt-checkbox-disabled-color);
+  }
+
+  &[disabled="true"] label {
+    color: var(--watt-checkbox-disabled-color);
+  }
+
+  label {
+    display: grid;
+    grid-template-columns: 1em auto;
+    gap: 0.5em;
+    align-items: center;
+  }
```

Comment:
nitpick: let's change the mouse cursor to hand on hover, indicating that the element can be clicked.

nitpick: on double click, the label text gets selected. Can see that Material is using `user-select: none;` to disable that. Should we add it too?